### PR TITLE
Add 1 case of cpu management of numa

### PR DESCRIPTION
**Pull request by chloerh**
- Start vm with missing vcpus in NUMA topology

Signed-off-by: Haijiao Zhao <haizhao@redhat.com>

### CODE REVIEW

**Comment by chloerh** [comment]
```
# avocado run --vt-type libvirt vcpu_fill
JOB ID     : 8c0a747967c07f36938722fcaadcbbcfa8f81f3c
JOB LOG    : /root/avocado/job-results/job-2020-12-02T18.49-8c0a747/job.log
 (1/2) type_specific.local.guest_numa.misc_test.vcpu_fill.set_to: PASS (19.45 s)
 (2/2) type_specific.local.guest_numa.misc_test.vcpu_fill.set_not_to: PASS (20.29 s)
RESULTS    : PASS 2 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB TIME   : 40.08 s
```

**Comment by chloerh** [comment]
@Yingshun would you like to review?

File: libvirt/tests/cfg/numa/guest_numa.cfg [hunk 1]
```diff
@@ -124,3 +124,21 @@
                      memnode_nodeset_0 = 0
                      memnode_cellid_0 = 0
                      memnode_mode_0 = "strict"
+        - misc_test:
+            variants:
+                - vcpu_fill:
+                    case = 'vcpu_fill'
+                    vcpu_num = 20
```
Comment (Yingshun) [inline libvirt/tests/cfg/numa/guest_numa.cfg:131]: Is it necessary to check host cpu number before these tests? I mean if the cpu number of the host is less than 20, what will happen?

Comment (dzhengfy) [reply libvirt/tests/cfg/numa/guest_numa.cfg:131]: Many cpu numbers on x86_64 or other arches machines are less than 20, so if you want the case to run on all platform, please judge the cpu numbers firstly.

Comment (dzhengfy) [inline libvirt/tests/cfg/numa/guest_numa.cfg:131]: Many cpu numbers on x86_64 or other arches machines are less than 20, so if you want the case to run on all platform, please judge the cpu numbers firstly.

**Comment by chloerh** [comment]
Updated code and tested on x86 
```
# avocado run --vt-type libvirt vcpu_fill
WARNING:root:No python imaging library installed. Screendump and Windows guest BSOD detection are disabled. In order to enable it, please install python-imaging or the equivalent for your distro.
WARNING:root:No python imaging library installed. PPM image conversion to JPEG disabled. In order to enable it, please install python-imaging or the equivalent for your distro.
JOB ID     : db89018334a932afe0faa597c2c26adf00ad6ff1
JOB LOG    : /root/avocado/job-results/job-2020-12-23T17.11-db89018/job.log
 (1/2) type_specific.local.guest_numa.misc_test.vcpu_fill.set_to: CANCEL: Number of vcpus(20) is larger than number of cpus on host(8). (2.73 s)
 (2/2) type_specific.local.guest_numa.misc_test.vcpu_fill.set_not_to: CANCEL: Number of vcpus(20) is larger than number of cpus on host(8). (2.69 s)
RESULTS    : PASS 0 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 2
JOB TIME   : 5.72 s
```

**Review by Yingshun — APPROVED** [review]

**Review by dzhengfy — APPROVED** [review]
LGTM

**Pull request merged**